### PR TITLE
Fixes #17 AUCM dimension

### LINKED ISSUE #17: AUCM dimension
@Yuri05 I don't think we have the dimension for this (umol*min2/l). Any objects if I had this? 
We are returning first moment for AUC know for userdefiend pk and we need to have the right dimension

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Please review

File: OSPSuite.Dimensions.xml [hunk 1]
```diff
@@ -123,6 +123,27 @@
...
     </Dimension>
+    <Dimension name="AUCM (molar)" baseUnit="µmol*min²/l">
+      <BaseRepresentation lengthExponent="-3" massExponent="0" timeExponent="2" electricCurrentExponent="0" temperatureExponent="0" amountExponent="1" luminousIntensityExponent="0" />
+      <Units>
+        <Unit name="mmol*h²/ml" factor="36E8" />
+        <Unit name="µmol*h²/ml" factor="36E5" />
+        <Unit name="nmol*h²/ml" factor="36E2" />
+        <Unit name="pmol*h²/ml" factor="36E-1" />
+        <Unit name="mmol*h²/l" factor="36E5" />
+        <Unit name="µmol*h²/l" factor="36E3" />
```
Comment (Yuri05) [inline OSPSuite.Dimensions.xml]: `<Unit name="µmol*h²/l" factor="36E3" />`
If i calculated correctly the factor must be 36E**2**

Comment (msevestre) [reply OSPSuite.Dimensions.xml]: that;s correct. I updated this one twice

Comment (msevestre) [reply OSPSuite.Dimensions.xml]: good catch

**Pull request merged**